help/develop/function_results_cache.md

```diff
@@ -0,0 +1,83 @@
+# Caching function results
+
+Datagrok can cache the results of functions on both client and server sides.
```

Comment:
What I liked about the previous version is that it listed the benefits ("to improve query performance.")